### PR TITLE
Machine parsable working groups

### PR DESCRIPTION
**Pull request by rkoster**
This PR refactors all working groups to be machine-readable. This is in preparation for later generating github org permissions based on these files.

### CODE REVIEW

File: toc/working-groups/app-runtime-platform.md [hunk 1]
```diff
@@ -29,158 +29,192 @@ Provides operational components for the CF App Runtime, including those for appl
...
+  - cloudfoundry/routing-api
+  - cloudfoundry/routing-api-cli
+  - cloudfoundry/routing-perf-release
+  - cloudfoundry/cf-tcp-router
+  - cloudfoundry/routing-info
+  - cloudfoundry/gorouter
+  - cloudfoundry/route-registrar
+  - cloudfoundry/haproxy-boshrelease
+  - cloudfoundry/cf-networking-release
+  - cloudfoundry/cf-networking-helpers
```
Comment (geofffranks) [inline toc/working-groups/app-runtime-platform.md:216]: Can we add cloudfoundry/policy_client to this list?

Comment (rkoster) [reply toc/working-groups/app-runtime-platform.md:216]: Adding PR reference: https://github.com/cloudfoundry/community/pull/196

**Comment by stephanme** [comment]
The recently added https://github.com/cloudfoundry/community/blob/main/toc/working-groups/app-runtime-deployments.md is missing.

**Comment by rkoster** [comment]
@stephanme the app runtime deployments working group has been added to this PR. Have also rebased on main and applied all changes that have happened to the other working groups since this PR was initially created manually.

**Review by LeePorte — APPROVED** [review]

**Review by stephanme — APPROVED** [review]

**Review by emalm — APPROVED** [review]
LGTM! Let's get everything updated to this format before we make any more changes to the WG charter files.

**Pull request merged**